Library/Formula/td.rb

```diff
@@ -0,0 +1,49 @@
+require "language/go"
+
+class Td < Formula
+  homepage "https://github.com/Swatto/td"
+  revision 1
```

Comment:
If this is a new formula, it doesn't have a `revision`. The `revision` is an unwritten 0.